azkaban-common/src/test/java/azkaban/executor/JdbcExecutorLoaderTest.java

```diff
@@ -739,6 +739,21 @@ public void testExecutorInactivation() throws Exception {
     Assert.assertFalse(fetchedExecutor.isActive());
   }
 
+  /* Test Removing Executor */
+  @Test
+  public void testRemovingExecutor() throws Exception {
+    if (!isTestSetup()) {
+      return;
+    }
+
+    ExecutorLoader loader = createLoader();
+    Executor executor = loader.addExecutor("localhost1", 12345);
+    loader.removeExecutor("localhost1", 12345);
+
+    Executor fetchedExecutor = loader.fetchExecutor("localhost1", 12345);
```

Comment:
May be its safer to assert that it exists before and then assert that is doesn't after delete.